src/sourmash/cli/tax/classify.py

```diff
@@ -0,0 +1,68 @@
+"""classify genomes from gather results"""
+
+import argparse
+import sourmash
+from sourmash.logging import notify, print_results, error
+from sourmash.cli.utils import add_threshold_arg
+
+def subparser(subparsers):
+    subparser = subparsers.add_parser('classify')
+    subparser.add_argument(
+        '-g', '--gather-csv', nargs='*', default = [],
+        help='csvs from sourmash gather'
```

Comment:
```suggestion
        help='CSVs output by sourmash gather for this sample'
```
(or something similar)